test/nursery/test_autogluon_tabular.py

```diff
@@ -173,23 +175,31 @@ def test_get_features_dataframe(
 )
 @pytest.mark.parametrize("lag_indices", [[], [1, 2, 5]])
 @pytest.mark.parametrize("disable_auto_regression", [False, True])
+@pytest.mark.parametrize("path", [Path("test_autogluon_dir")])
+@pytest.mark.parametrize("ag_path_prefix", [None, Path("test_autogluon_dir")])
```

Comment:
There is no need to parametrize the test with these paths, I think: It would be better to use a temporary directory in tests, see https://docs.python.org/3.7/library/tempfile.html